manage.py

```diff
@@ -118,6 +97,85 @@ def analyze_all(self):
         )
         print('Batcher invocation successful!')
 
+    @staticmethod
+    def build():
+        """Build Lambda packages (saves *.zip files in terraform/)."""
+        lambda_build(TERRAFORM_DIR)
+
+    def deploy(self):
+        """Deploy BinaryAlert. Equivalent to test + build + apply + analyze_all."""
+        self.test()
+        self.build()
+        self.apply()
+        self.analyze_all()
+
+    def live_test(self):
+        """Upload an EICAR test file to BinaryAlert which should trigger a YARA match alert."""
+        bucket_name = '{}.binaryalert-binaries.{}'.format(
+            self._config['name_prefix'].replace('_', '.'), self._config['aws_region'])
+        test_filename = 'eicar_test_{}.txt'.format(uuid.uuid4())
+        s3_identifier = 'S3:{}:{}'.format(bucket_name, test_filename)
+
+        print('Uploading {}...'.format(s3_identifier))
+        bucket = boto3.resource('s3').Bucket(bucket_name)
+        bucket.put_object(
+            Body=EICAR_STRING.encode('UTF-8'),
+            Key=test_filename,
+            Metadata={'observed_path': test_filename}
+        )
+
+        print('File uploaded! Waiting for new Dynamo entry to appear...')
+        table_name = '{}_binaryalert_matches'.format(self._config['name_prefix'])
+        table = boto3.resource('dynamodb').Table(table_name)
+        eicar_sha256 = hashlib.sha256(EICAR_STRING.encode('UTF-8')).hexdigest()
+        dynamo_record_found = False
+        lambda_version = 0
+
+        for attempt in range(1, 11):
+            time.sleep(5)
+            print('\t[{}/10] Scanning DynamoDB:{}...'.format(attempt, table_name))
+            items = table.query(
+                Select='ALL_ATTRIBUTES',
+                Limit=1,
+                ConsistentRead=True,
+                ScanIndexForward=False,  # Sort by LambdaVersion descending (e.g. newest first).
+                KeyConditionExpression=Key('SHA256').eq(eicar_sha256),
+                FilterExpression=Attr('S3Objects').contains(s3_identifier)
+            ).get('Items')
+            if items:
+                print('\tSUCCESS: Dynamo entry found!\n')
+                dynamo_record_found = True
+                lambda_version = items[0]['LambdaVersion']
+                pprint.pprint(items[0])
+                break
+            elif attempt == 10:
+                print('\nFAIL: Entry not found')
+
+        print('\nRemoving EICAR test file from S3...')
+        bucket.delete_objects(Delete={'Objects': [{'Key': test_filename}]})
+
+        print('Removing Dynamo EICAR entry...')
+        table.delete_item(Key={'SHA256': eicar_sha256, 'LambdaVersion': lambda_version})
+
+        if dynamo_record_found:
+            print('\nLive test succeeded! Verify the alert was sent to your SNS subscription(s).')
+        else:
+            sys.exit('\nLive test failed!')
```

Comment:
I'm not really sure what to put. Since this is an end-to-end test, literally anything could be wrong. BinaryAlert might not have deployed correctly, S3/SNS/SQS/Lambda/Dynamo might be down, the YARA rules could be broken, etc.

My thought is to add a `Troubleshooting` section to the upcoming documentation, and then we could link to that here?